SPECS-EXTENDED/ldns/ldns.spec

```diff
@@ -209,23 +184,18 @@ export CFLAGS CXXFLAGS LDFLAGS
   %global disable_dane_ta --disable-dane-ta-usage
 %endif
 
-%global common_args \\\
-  --disable-rpath \\\
-  %{enable_gost} %{enable_ecdsa} %{enable_eddsa} %{?disable_dane_ta} \\\
-  --with-ca-file=/etc/pki/tls/certs/ca-bundle.trust.crt \\\
-  --with-ca-path=/etc/pki/tls/certs/ \\\
-  --with-trust-anchor=%{_sharedstatedir}/unbound/root.key \\\
-  --disable-static \\\
-
-
 %if 0%{with python3}
 pushd %{pkgname}_python3
 %else
 pushd %{pkgname}
 %endif # with python3
 
 %configure \
-  %{common_args} \
+  --disable-rpath \
+  %{enable_gost} %{enable_ecdsa} %{enable_eddsa} %{?disable_dane_ta} \
+  --with-ca-file=/etc/pki/tls/certs/ca-bundle.trust.crt \
```

Comment:
Do we use these cert paths enough in specs that we might want to create a macro for it? Not pertinent to this PR, just wondering since you're our ca-certificates maintainer.

Reply:
I agree, it is a good idea to have a macro for our cert locations, since we already use them in more than one place. And like you said, not part of this change, but I'll look into doing separately.